apps/api-frontend/src/app/app.css

```diff
@@ -15,6 +16,16 @@
   display: inline-block;
 }
 
+.app .footer {
```

Comment:
this doesn't show up as a floating footer 
https://screenshot.googleplex.com/103M0yjZW0b.png

You probably want to give the footer some sort of background so it doesn't overlap with text, and make it "float" by giving it a fixed position:
https://stackoverflow.com/questions/13609006/floating-footer-always-on-the-bottom-and-visible

Right now, scrolling down the results page will show the footer in the middle of the page, instead of keeping it at the bottom.
Making it a fixed position will fix that, and adding a background (e.g. white background, maybe with a shadow on the page) will make sure it doesn't overlap with the text